go/vt/vtgate/semantics/early_rewriter_test.go

```diff
@@ -536,6 +536,11 @@ func TestHavingColumnName(t *testing.T) {
 		expSQL:  "select id, sum(t1.foo) as foo from t1 having sum(foo) > 1",
 		expDeps: TS0,
 		warning: "Column 'foo' in having clause is ambiguous",
+	}, {
+		sql:     "select id, sum(t1.foo) as foo from t1 having custom_udf(foo) > 1",
+		expSQL:  "select id, sum(t1.foo) as foo from t1 having custom_udf(foo) > 1",
+		expDeps: TS0,
+		warning: "Column 'foo' in having clause is ambiguous",
 	}, {
```

Comment:
can you add one more test with custom udf similar to 
`select id, sum(foo) as sumOfFoo from t1 having sumOfFoo > 1`